index.bs

```diff
@@ -231,57 +231,57 @@ Given a task |task|, perform the following algorithm:
 
 2. Let |destinationRealms| be an empty set.
 
-3. Determine the set of JavaScript Realms to which reports will be delivered:
+3. Determine the set of <a href="https://tc39.github.io/ecma262/#sec-code-realms">JavaScript Realms</a> to which reports will be delivered:
```

Comment:
The more "Bikeshed" way of doing this would be to add to the link block, e.g. https://github.com/whatwg/streams/blob/51227372cc84846bdcf68312724c4cac6a4b9e58/index.bs#L25. No big deal.

Reply:
Done, also removed other 'a href' in favor of urlPrefix.